apps/sharing/helpers.py

```diff
@@ -12,7 +12,6 @@ def sharing_widget(context, obj, condensed=False):
 
     c.update({
         'condensed': condensed,
-        'base_url': obj.share_url(),
```

Comment:
The template already does this:
```
data-base-url="{{ obj.share_url() }}?service="
```

So adding `base_url` to the context appears to be useless.